webui/react/package.json

```diff
@@ -33,9 +33,9 @@
     "portable-fetch": "^3.0.0",
     "pretty-bytes": "^5.4.1",
     "query-string": "^6.13.5",
-    "react": "^16.14.0",
-    "react-dom": "^16.14.0",
-    "react-helmet": "^6.1.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```

Comment:
decided to upgrade here to see how it went and surprisingly smooth because everything was resolved during the `Strict Mode` application.

[React 17 Migration Guide](https://reactjs.org/blog/2020/10/20/react-v17.html)